looker-upload-artifact-registry.sh

```diff
@@ -19,7 +19,9 @@
 # $1 is the sub-project directory
 # $2 is the artifact ID
 # $3 is the version
-function artifact_registry_upload {
+
+# Note: Redeployment of the same versioned artifact (same name, repeated uploads) is prohibited.
```

Comment:
did you determine if this is for both snapshots and releases or just one or the other?

Reply:
Only the release repo prevents repeated uploads, the snapshot one is fine